scopes/component/snapping/tag-cmd.ts

```diff
@@ -67,23 +67,27 @@ if patterns are entered, you can specify a version per pattern using "@" sign, e
 [${Object.keys(IssuesClasses).join(', ')}]
 to ignore multiple issues, separate them by a comma and wrap with quotes. to ignore all issues, specify "*".`,
     ],
-    ['I', 'ignore-newest-version', 'ignore existing of newer versions (default = false)'],
+    [
+      'I',
+      'ignore-newest-version',
+      'allow tagging even when the component has newer versions e.g. for hotfixes (default = false)',
+    ],
     [
       '',
       'fail-fast',
-      'stop pipeline execution on the first failed task (by default a task is skipped only when its dependent failed)',
+      'stop pipeline execution on the first failed task (by default a task is skipped only when its dependency failed)',
```

Comment:
same as with "snap" command. should be "dependent".

Reply:
see my comment there - dont think you're right...

Reply:
yup. I was wrong